templates/tags/list.html

```diff
@@ -14,7 +14,8 @@
                 {% if config.extra.compact_tags %}
                     {# Shows the number of posts per tag as a superscript #}
                     <a href="{{ term.permalink | safe }}"
-                       aria-label="{{ term.name }} – 
+                       aria-label="{{ term.name }} &nbsp;– &nbsp;
+                       {{- macros_translate::translate(key="post_pre", default="", language_strings=language_strings) -}}&nbsp;
```

Comment:
✅ `post_pre` translation is good
🛠️ there might be too many spaces and `&nbsp;`. Will need to review with final PR